### PR TITLE
BAU: Simplify S3 bucket creation. Allows choice of policies.

### PR DESCRIPTION
**Pull request by carinadigital**

### CODE REVIEW

File: roles/ec2-app/tasks/destroy_stack.yml [hunk 1]
```diff
@@ -209,3 +209,13 @@
 
 - name: Destroy security groups
   include: destroy_sgs.yml
+
+# Commented out intentionally. We want to keep s3 buckets and data between our builds
```
Comment (brettminnie) [inline roles/ec2-app/tasks/destroy_stack.yml:213]: why not add this in a switch type scenario

```
- block:
     - name: Destroy main s3 resources
    #destroy stuff here
    when: destroy_buckets is defined and destroy_buckets
```

**Review by brettminnie — COMMENTED** [review]
See comment on destroy_stack.yml

**Review by brettminnie — CHANGES_REQUESTED** [review]
Arrrgh! Stupid github :dagger:

**Review by tim-mcarthur-moj — CHANGES_REQUESTED** [review]
all yaml files should have --- before the vars are defined.

**Review by tim-mcarthur-moj — APPROVED** [review]

**Pull request merged**